### PR TITLE
Added support for exception event.

### PR DESCRIPTION
**Pull request by andr3medeiros**
Usage

```javascript
$analytics.exceptionTrack({
	description: exception.message + ' (cause: ' + exception.cause + ')',
	fatal: true,
	appId: APP.appId,
	appName: APP.appName,
	appVersion: APP.appVersion
});
```

### CODE REVIEW

**Comment by lunks** [comment]
I think `expectionTrack` makes more sense in the API. Could you please change the name?

**Comment by andr3medeiros** [comment]
Agreed and done @lunks.

File: src/angulartics-ga.js [hunk 1]
```diff
@@ -92,5 +92,21 @@ angular.module('angulartics.google.analytics', ['angulartics'])
 
   });
 
+  $analyticsProvider.registerExceptionTrack(function (properties) {
+    if (!properties || !properties.appId || !properties.appName || !properties.appVersion) {
+        console.error('Must set appId, appName and appVersion.');
+        return;
+    }
+
+    if(!properties.fatal) {
```
Comment (lunks) [inline src/angulartics-ga.js]: This way I will never be able to send `fatal` as `false`, which I don't think is what we should to here. Maybe change to
`if (properties.fatal == undefined) {`?

**Comment by andr3medeiros** [comment]
You're rigth. Force of habit.

**Comment by luisfarzati** [comment]
@andr3medeiros looks good for me now, would you please rebase? And also please add documentation in the README regarding usage.

**Comment by infomofo** [comment]
Is this on the roadmap for merging into master? It would be great to be able to track exceptions in GA, NewRelic, and other platforms in one place.

**Comment by ProLoser** [comment]
This looks good to me @angulartics/core should we merge this in?

**Comment by luisfarzati** [comment]
Well, there's no documentation, but it could be an undocumented feature and care about the doc later.

**Comment by ProLoser** [comment]
Conflicts :( @andr3medeiros any chance you can rebase this?

**Comment by andr3medeiros** [comment]
@ProLoser I will take a look this afternoon, probably 13pm (America/Sao_Paulo timezone).

**Comment by andr3medeiros** [comment]
@ProLoser, just did a pull request to angulartics/angulartics-google-analytics (https://github.com/angulartics/angulartics-google-analytics/pull/12) with the implementation. Also, I rebased my fork. Please try again.

**Comment by ProLoser** [comment]
@andr3medeiros I'm sorry about the hassle I'm just soooo bogged down lately. Can I add you to this repo and the ga repo so you can merge and do a proper release?

**Comment by bluesliverx** [comment]
+1, I'm excited to see this in core.

**Comment by timelf123** [comment]
@andr3medeiros I'd love to merge this but it needs a little love to make the tests pass, since GA has been split to it's own repository

**Comment by timelf123** [comment]
ping @andr3medeiros are you able to fix this up so we can merge?

**Comment by Emeegeemee** [comment]
Could we get this merged, it seems to have broken angulartics-google-analytics because the implementation was merged without the core having the register function yet. If not can we revert the implementation commit?

**Comment by timelf123** [comment]
My bad on merging the GA stuff early -- I had forgotten this one was sitting in limbo. If'm having some troubles with the tests because GA is now in it's own repo.

I'll push up a hotfix in the next hour or so that will either revert the GA implementation or add the register fn to the core

**Comment by timelf123** [comment]
1.0.2 solves this. For some reason 1.0.1 had already been published so I had to unpublish and use 1.0.2...

The tests are not included in that commit because they were not working (this code was written when GA was included in this core repo)

**Comment by andr3medeiros** [comment]
I am very sorry for my absence. I was away due personal issues.

**Comment by timelf123** [comment]
No worries, glad to have you back @andr3medeiros

**Pull request closed without merging**